src/example.mdx

```diff
@@ -1,4 +1,7 @@
-# Hello
+export const meta = {
+    title: "Hello",
+    published: "2022-01-04",
+};
```

Comment:
In actual, tsc does not check this `meta` typing.

https://github.com/microsoft/TypeScript/issues/36440